src/Components/Reports/SystemsPDFReport.js

```diff
@@ -1,51 +1,72 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { DownloadButton } from '@redhat-cloud-services/frontend-components-pdf-generator';
-import { PDF_REPORT_PER_PAGE, SYSTEMS_PDF_REPORT_TITLE } from '../../constants';
+import { PDF_REPORT_PER_PAGE, REPORT_NOTIFICATIONS, SYSTEMS_PDF_REPORT_TITLE } from '../../constants';
 import { fetchSystems } from '../../Utilities/api';
 import { formatData, generateFilterText, getSystemsReportFileName } from './Util';
 import propTypes from 'prop-types';
 import { SystemsTablePage } from './Common/SystemsTablePage';
 import { SystemsFirstPage } from './Common/SystemsFirstPage';
+import { useNotification } from './Common/useNotification';
 
-const generateSystemsPDFReport = async (filters, orderBy, orderHow) => {
+export const DownloadSystemsPDFReport = ({ filters, orderBy, orderHow, ...props }) => {
+    const reportFileName = getSystemsReportFileName();
+    const [addNotification, clearNotifications] = useNotification();
```

Comment:
Sure @kgaikwad. The reason I created a seperate custom hook is in case we want to use same at some other place but after looking into to code again I think its better to use it directly from the library since we have separated the object (for success, error etc) so its not much of the code. I will push the changes in the latest commit. 💯